### DISCUSSION TITLE
AADSTS50020 error when trying to login with microsoft "live" account

### DISCUSSION THREAD

**Discussion by sinuheshinbr** [nextauthjs/next-auth#7448] [Help]
### Question 💬

I configured Azure AD auth following the docs. When I access with my microsoft account that is part of the organization it works as expected. When I try to use another outlook account I receive the following error:

AADSTS50020: User account '<my-email>@outlook.com' from identity provider 'live.com' does not exist in tenant 'Diretório Padrão' and cannot access the application '<application-id>'(<name-of-the-application>) in that tenant. The account needs to be added as an external user in the tenant first. Sign out and sign in again with a different Azure Active Directory user account.

### How to reproduce ☕️

Add AzureAD provider following the documentation and try to log in with an outlook account that is not part of the same organization

### Contributing 🙌🏽

Yes, I am willing to help answer this question in a PR

**Comment by davewasthere** [comment]
I suspect you've got the tenant id set to a guid in your environment variables.
Either omiting it completely, or setting it explicitly to common, will let you log in with both organizational accounts, and consumer accounts.

e.g. AZURE_AD_TENANT_ID=common
